charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus once per calendar year. There's no term limit; the same individual can be reappointed as chair.
+
+Meeting time and agenda should be announced by the chair at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
+
+## Scope
+
+The output of the Interop project is metrics designed to measure and improve the interoperability of the web platform.
+
+These metrics are expected to be comprised of:
+
+- Groups of automated tests that can be used to automatically compute an interoperability score for specific platform features in implementations.
+- Work to improve the interoperability of the platform that cannot be measured through test pass rates and is scored by consensus decision.
+
+The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
+
+Initially the team will produce one set of interop metrics per year. This may be changed in the future by consensus decision.
+
+If at any time the team is unable to reach consensus on which features to include in its metrics, or the set of features which has consensus is considered insufficient to fulfill the goals of the team, the effort will be paused until the situation can be resolved.
```

Comment:
In theory if we had >1 metric, the default would be to stop only those metrics for which we didn't have consensus. It's not really clear if this paragraph is intending to be specific to the case where there are zero agreeable things or not.

Reply:
Since @boazsender also had questions about this paragraph, I'll just remove it instead of trying to spell out exactly the way things can eventually go wrong.